bptestrunner/bluepill_batch_test.bzl

```diff
@@ -29,9 +29,9 @@ def _bluepill_batch_test_impl(ctx):
 
         #test_plan
         test_plan = struct(
-            test_host = test_host.basename.rstrip(
+            test_host = test_host.basename.split(
                 "." + test_host.extension,
```

Comment:
Shouldn't this be just `split(".")`?

Reply:
That was another option. Assuming there isn't going to be a `.` in the test host's name. The reason why I went this way was it was more clear what we are splitting against, for instance we are specifically splitting on the the `.` + extension so in case there is a `.` in the test host which there likely shouldn't/won't be this will not split on it.

I'm open to splitting JUST on the `.` though, just thought this made it a little more future proof

Reply:
Hmm... you're right... I think your way is safer.

Another option would be to depend on Skylib and use [`paths.replace_extension()`](https://github.com/bazelbuild/bazel-skylib/blob/6e30a77347071ab22ce346b6d20cf8912919f644/docs/paths_doc.md#pathsreplace_extension), but I'm ok with how you're doing it.